src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FFIDMetadataRouteSpec.scala

```diff
@@ -58,6 +59,16 @@ class FFIDMetadataRouteSpec extends AnyFlatSpec with Matchers with TestRequest w
     checkFFIDMetadataExists(response.data.get.addFFIDMetadata.fileId)
   }
 
+  "addFFIDMetadata" should "set the file status to password protected found when a password protected file is found" in {
+    runTestMutation("mutation_password_protected", validBackendChecksToken("file_format"))
+    getFileStatusResult(defaultFileId, FFID) should equal(PasswordProtected)
+  }
+
+  "addFFIDMetadata" should "set the file status to success when there are no password protected files found" in {
+    runTestMutation("mutation_alldata", validBackendChecksToken("file_format"))
+    getFileStatusResult(defaultFileId, FFID) should equal(Success)
+  }
+
```

Comment:
Is it worth adding a test where we've got one match which shows password protected and another that shows a valid puid? I got tripped up with multiple matches on the last one and it might break later as we add more invalid puids